cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/RemovedObjectParsingStrategy.java

```diff
@@ -22,9 +26,14 @@
 
     @Override
     protected List<CfChange> doParse(Change change) {
-        return Arrays.asList(new CfRemovedObject(change.getAffectedObject().get(),
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
+                CfRemovedObject.class.getSimpleName(), "with object", change.getAffectedObject().get());
+        List<CfChange> cfChanges = Collections.singletonList(new CfRemovedObject(change.getAffectedObject().get(),
                 "",
                 extractPathFrom(change)));
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
```

Comment:
I would recommend to use `debug` logging here to prevent long and confusing output